responses/05_map-events.md

```diff
@@ -0,0 +1,39 @@
+# Events
+
+Alright - you've gotten a project chosen, and you've laid out the steps for your learners. Next, we're going to get into something new with Learning Lab: events! (You can learn more detail about this in [the documentation](https://lab.github.com/docs/events).)
+
+An `event` is the webhook that the learner triggers. Events tell Learning Lab that it should do something. [You can see all of the events in GitHub's documentation](https://developer.github.com/v3/activity/events/types/). Some of the most common examples are `pull_request.synchronize` or `issue.comment`. 
+
+### Map behaviors to events
+
+How can each step translate to a GitHub event? Having too many of the same event may be a bad signal. Make sure that events represent things that you're trying to teach.
+
+For example, you may want to show a lot of information to the learner, and then have them close the issue to signify they've read it. That may make sense for one or two steps in your course. But, imagine going through a whole course like that. It isn't actually checking if the learner read - it's checking if the learner knows how to close issues!
+
+### Choose the right events
+
+Try to choose events that correspond directly to what you want the learner to do. If you're trying to teach the learner to import a `npm` module into a `package.json` file, that commit should be the event.
+
+| Behavior | Events |
+| ------------- | ------------- |
+| What can I observe that confirms the user demonstrated the skill or knowledge? | What event triggers are available through GitHub? |
+| Write a function (commit) | [`pull_request.synchronize`](https://developer.github.com/v3/activity/events/types/#pullrequestevent) |
+| Solve a merge conflict (commit | [`push`](https://developer.github.com/v3/activity/events/types/#pushevent) |
+| Open a pull request | [`pull_request`](https://developer.github.com/v3/activity/events/types/#pullrequestevent) |
+| Implement a test | [`status`](https://developer.github.com/v3/activity/events/types/#statusevent) |
+| Comment on an issue | [`issue_commented.created`](https://developer.github.com/v3/activity/events/types/#issuecommentevent) |
+
+## Step 5: Map learning objectives to events
+
+Next, your job is to map your learning objectives to events.
+
+Remember the steps you wrote earlier? Let's find the corresponding events. You'll see some are already done for the examples, but you can focus on your own.
+
+### :keyboard: Activity: Map the learning objectives you wrote to specific events from GitHub webhooks
+
+1. Take a look through [GitHub's documentation for event triggers](https://developer.github.com/v3/activity/events/types/).
+2. [Edit the `config.yml` file]({{ url }}) on line 29 and make note of the event trigger that matches your first objective.
```

Comment:
@brianamarie since the PR is created when I join the course, I can't see what I wrote here for steps. That makes it a little challenging to remember what to do next. Again, could we instruct them to make this change and then open the PR?

Reply:
@crichID I'm open to having the user open their own PR, but I had thought that [lines 114-116 in the config, where master is merged into this branch](https://github.com/githubtraining/write-a-ll-course/pull/40/files#diff-0f1be8eec416e71c28b41efb0544df26R114-R116), would have prevented this. When I go through the course after these edits, I will keep an eye out for this. 

If the changes _were_ present, would that solve the problem? Or are there other reasons that I'm missing that would make the flow of having the user open the PR easier?